### ISSUE TITLE
not autoalias after uninstall a ruby version

### ISSUE THREAD

**Issue by bachue** [tpope/rbenv-aliases#8]
Why not re-alias automatically after uninstall a ruby version?

**Comment by jasonkarns** [comment]
This logic is already there: https://github.com/tpope/rbenv-aliases/blob/155dc4404a2eac0a134d6357ad2d0b7ea302e302/bin/rbenv-alias#L33-L41

Sorta. `rbenv alias --auto` without specifying a version already cleans up any dangling aliases. However, 

``` sh
$ rbenv alias <missing_version> --auto
Couldn't find any versions for <missing_version>
```

Unfortunately, we can't just call `rbenv alias --auto` on uninstall because that could add/alter aliases to other versions that shouldn't be affected by the uninstalled version.

**Comment by jasonkarns** [comment]
for reference, I've implemented this feature in nodenv-aliases [here](1f73427bd89796e4aefb9ec48a515c5e1ed0d8f6)

Hoping to do a PR here soon
